vlib/v/gen/c/testdata/inline_asm_rv64_displacement_with_base_addressing.vv

```diff
@@ -0,0 +1,57 @@
+// vtest vflags: -cc riscv64-linux-gnu-gcc -gc none
+// vtest runner: qemu-riscv64
+// vtest runner_flags: -L /usr/riscv64-linux-gnu/
+
+@[packed]
+struct SomeStruct {
+	element_0 u64
+	element_1 u64
+}
+
+fn main() {
+	if check_displacement_with_base() == true && check_composite_displacement_with_base() == true {
+		println('pass')
+	} else {
+		println('fail')
+	}
```

Comment:
It will be more readable, and easier to diagnose the failed condition on the CI remotely, if you have this instead:
```suggestion
	if check_displacement_with_base() {
		println('pass displacement_with_base')
	} else {
		println('fail displacement_with_base')
	}
	if check_composite_displacement_with_base() {
		println('pass composite_displacement_with_base')
	} else {
		println('fail composite_displacement_with_base')
	}
```

Reply:
(the code is more verbose, but also easier to follow)